README.md

```diff
@@ -85,6 +85,13 @@ The helper to load migrations `require_migration` is defined in the `migration_d
 
 Use `rake db:migrate:squash` to remove all your old migrations and generate one migration with the current database schema. You don't have to run migrations after this because the generated migration will have the latest database version.
 
+## Skipping data migration on test environment
+
+If you run migration in test environment, data migration might try to add same data that you already have in seeds.
```

Comment:
`On performing migrations in test environment, a data migration might try to add same data that is already added by seeds. In that case the migration fails with a duplication error.`